pkg/web/api/v1/transaction.go

```diff
@@ -96,6 +96,11 @@ type Rejection struct {
 	RequestRetry bool   `json:"request_retry"`
 }
 
+type Repair struct {
+	Error    *Rejection
+	Envelope *Envelope
+}
```

Comment:
This payload will allow you to display the repair error message to the user when you're displaying the repair side of things.